### PR TITLE
Fix demo error

### PR DESCRIPTION
**Pull request by mwonng**
- fix npm/yarn install permission problem with moltin/sdk.
- fix demo error from API response.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "start": "next start"
   },
   "dependencies": {
-    "@moltin/sdk": "github:ynnoj/js-sdk",
```
Comment (mwonng) [inline package.json:10]: This dependency looks are very old one and now it has no permission for me to run `npm install` or `yarn install`.
Not sure if the version is right with Next.js but using the latest one.

**Review by notrab — APPROVED** [review]

**Comment by KalleVuorjoki** [comment]
Run into same problem. Had to downgrade node.js to v8.12.0 in order to `npm install` to work.

**Comment by notrab** [comment]
@KalleVuorjoki @mwonng I'll be able to get this merged if we can use the memory storage adapter. This will prevent node-localstorage from trying to write to disk.

https://github.com/moltin/js-sdk#custom-storage

If you are able to make the changes I'll get this merged and the demo link updated 😄 

Thanks for your contribution ❤️

**Comment by notrab** [comment]
Hi @KalleVuorjoki @mwonng 

Apologies for the separate PR but I went ahead and fixed the issue while debugging another issue. I merged https://github.com/moltin-examples/nextjs-demo-store/pull/8 which contains the same fixes as this PR.

Shoot me an email to `jamie.barton@moltin.com` and I'll send you some @moltin swag as a thankyou 😅

**Comment by mwonng** [comment]
> Hi @KalleVuorjoki @mwonng
> 
> Apologies for the separate PR but I went ahead and fixed the issue while debugging another issue. I merged #8 which contains the same fixes as this PR.
> 
> Shoot me an email to `jamie.barton@moltin.com` and I'll send you some @moltin swag as a thankyou 😅

Sorry Jamie, I broken my elbow last month still in the life with right arm only. I cant update for now.

**Pull request closed without merging**